src/k8s-engine/minikube.js

```diff
@@ -238,7 +239,7 @@ class Minikube extends EventEmitter {
 
       bat.on('exit', code => {
         this.clear();
-        if (code === 0 || code === undefined || code === null) {
+        if (code === 0 || code === undefined || code === null || originalState === K8s.State.ERROR) {
```

Comment:
Does this do the right thing? If I'm reading this right, if we _start_ from an error state, and then fail to stop, we just assume we've stopped successfully? Not sure that's correct, since nothing actually says we did anything useful here…
(Also, it's unclear to me why this is in _this_ PR.)